test/extensions/bootstrap/wasm/test_data/speed_cpp.cc

```diff
@@ -69,6 +73,46 @@ void grpc_service1000_test() {
   }
 }
 
+void modify_metadata_test() {
+  auto path = getRequestHeader(":path");
+  addRequestHeader("newheader", "newheadervalue");
+  auto server = getRequestHeader("server");
+  replaceRequestHeader("server", "envoy-wasm");
+  replaceRequestHeader("envoy-wasm", "server");
+  removeRequestHeader("newheader");
+}
+
+void modify_metadata1000_test() {
+  for (int x = 0; x < 1000; x++) {
+    auto path = getRequestHeader(":path");
+    addRequestHeader("newheader", "newheadervalue");
+    auto server = getRequestHeader("server");
+    replaceRequestHeader("server", "envoy-wasm");
+    replaceRequestHeader("envoy-wasm", "server");
+    removeRequestHeader("newheader");
+  }
+}
+
+void json_serialize_test() {
+  const std::string configuration = R"EOF(
```

Comment:
It would be nice if we could get something larger and more "representitive".  Perhaps @bianpengyuan has some suggestion.